src/main/java/wolox/training/exceptions/responses/BookAlreadyOwnException.java

```diff
@@ -1,9 +1,5 @@
 package wolox.training.exceptions.responses;
 
-import org.springframework.http.HttpStatus;
-import org.springframework.web.bind.annotation.ResponseStatus;
-
-@ResponseStatus(HttpStatus.BAD_REQUEST)
 public class BookAlreadyOwnException extends RuntimeException{
```

Comment:
You can have a single BookErrors class that extends from Exception, this way you create a method for each type of error, where the method looks something like this:
    public void bookNotFound () {
throw new ResponseStatusException (HttpStatus.NOT_FOUND, this.getMessage ());
}
Where the first parameter is the type of error (404, 403, etc) and the second the error message you want to send